src/Component/LoginForm/LoginState.d.ts

```diff
@@ -0,0 +1,9 @@
+type LocalInitialStateType = {
```

Comment:
Loved this implicit Type specification. Learnt this new thing!

Reply:
Thanks!!